index.html

```diff
@@ -26,20 +22,39 @@
       <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
-
 </head>
 
-  <body>
-
-    <div class="posts"></div>
-
-
+  <body class ="lista">
+  <!-- Start Bar -->
+  <p class="cblog">Cognits Blog</p>
+  <p class="home">Home</p>
+  <p class="search">Search</p> 
+  <hr class="separators">
+
+  <!-- Post Title -->
+  <div class="parte2">
+  <p class="searching">Buscar  » <input name="buscador" id="buscador" type="text" /></p>
+  <p class="blank">ESTO ES PARA HACER UN ESPACIO EN BLANCO</p>
+  <p class="nota1">Cognits Blog</p>
+  <p class="nota2">El proyecto para que aprendan a usar HTML, CSS y Javascript.</p>
+  </div>
+
+  <!-- Area of Post -->
+  <div class = "menu"></div>
+
+  <!-- Boton Older Post -->
+  <div class="boton1">
+    <a class="but" href="https://github.com/KAR2014"> Older Post <span>→</span></a>
+  </div>
+    <!-- Credits -->
+    <hr class="separators">
+    <p class="creditos">Copyright© My Website 2015  ******  Kevin Roberto - KAR2014 </p>
     <!-- Bootstrap core JavaScript
```

Comment:
Great work documenting your code. You can think of the comments in a code as the intentions or thoughts of the programmer in the moment, which can clarify things when someone else reads it. 
:100: